extract.py

```diff
@@ -7,17 +7,6 @@
 NO_OF_PLANTS = 51
```

Comment:
Think you may have to make sure they are the same

Reply:
as in the function I deleted? Yeah true, I can add it back just in case for now

Reply:
Misunderstood it sorry, will approve now